os2borgerpc_client/bin/os2borgerpc_register_in_admin

```diff
@@ -1,24 +1,24 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 import sys
-import urlparse
+import urllib.parse
 
-from bibos_utils.bibos_config import BibOSConfig
-from bibos_client.admin_client import BibOSAdmin
+from os2borgerpc.client.config import OS2borgerPCConfig
+from os2borgerpc.client.admin_client import OS2borgerPCAdmin
 
 
-config = BibOSConfig()
+config = OS2borgerPCConfig()
 data = config.get_data()
 
 mac = data['mac']
 hostname = data['hostname']
 site = data['site']
 distribution = data['distribution']
-admin_url = data.get('admin_url', 'http://bibos.magenta-aps.dk/')
+admin_url = data.get('admin_url', 'http://os2borgerpc.magenta-aps.dk/')
```

Comment:
Okay we need to add this as an alias to our production server...

Reply:
Yeah, this one is weird. We _already_ don't have a system called `bibos.magenta-aps.dk`...

(`register_new_os2borgerpc_client.sh` will actually always override this default value, so we never use it. Should we do something about that?)

Reply:
What _is_ this server called now? (And shouldn't the URL be `https` in all cases?)

Reply:
Yeah, this should probably be fixed up a bit.